### PR TITLE
[DOCS-6349] ACS 7.1 Stack updates

### PR DESCRIPTION
**Pull request by anxumalo**

### CODE REVIEW

**Review by oliveralfresco2018 — APPROVED** [review]
LGTM cheers.

File: content-services/latest/support/index.md [hunk 1]
```diff
@@ -11,28 +11,27 @@ Choose a combination of products to build your own Supported Stack. If anything
...
 | **Databases** | |
 | MySQL 8 | `mysql-connector-java-8.0.22.jar` |
 | MySQL 5.7.23 | `mysql-connector-java-5.1.40-bin.jar` |
 | MS SQL Server 2019 | `mssql-jdbc-8.4.1.jre11.jar` |
 | MS SQL Server 2017 | Microsoft JDBC Driver `7.2.2.jre11` for SQL Server. See [JDBC Driver](https://docs.microsoft.com/en-us/sql/connect/jdbc/system-requirements-for-the-jdbc-driver?view=sql-server-ver15){:target="_blank"} for supported versions. |
-| Oracle 19c | `Ojdbc8.jar – 12.2.0.1` |
-| Oracle 12c | `Ojdbc8.jar – 12.2.0.1` |
-| PostgreSQL 13.1 |`Postgresql-42.2.19.jar`|
-| PostgreSQL 12.4 |`Postgresql-42.2.18.jar` |
+| Oracle 19c | `Ojdbc8.jar – 19.3.0.0` |
```
Comment (alandavis) [inline content-services/latest/support/index.md:28]: I would add 12.2.0.1 to the list of Ojdbc8.jar versions, as we had some problems with 19.<anything>. The 19.3.0.0 version probably is fin however. https://alfresco.atlassian.net/browse/ACS-1841 was raised to investigate further.

File: content-services/latest/support/index.md [hunk 2]
```diff
@@ -11,28 +11,27 @@ Choose a combination of products to build your own Supported Stack. If anything
...
+| PostgreSQL 13.3 | `Postgresql-42.2.19.jar` |
+| PostgreSQL 12.7 | |
+| PostgreSQL 12.4 | `Postgresql-42.2.18.jar` |
+| PostgreSQL 11.12 | |
 | PostgreSQL 11.7 | `Postgresql-42.2.6.jar` |
```
Comment (alandavis) [inline content-services/latest/support/index.md:33]: 11.7 should not be in the list

**Review by alandavis — APPROVED** [review]
I have added a few small changes.

**Comment by anxumalo** [comment]
@alandavis Thanks for the clarifications - updates applied as requested

**Pull request merged**